Webex Connect Flows/v3.3/Sample/Usage Of Surveys in Flows/README.md

```diff
@@ -0,0 +1,87 @@
+
+
+# Survey Sample Flows Overview :
+- This flow demonstrates the usage of WXM node using FacebookTaskCloseFlow and LiveChatTaskCloseFlow.
+- When a flow administrator needs to send survey messages or survey links to customers, these flows can be utilized.
+
+## Setting up the flow with a Wxm node.
```

Comment:
Please keep the node name consistent (as per connect node name) across all places, WxM WXM Wxm

Reply:
Done